cinema/models.py

```diff
@@ -1,10 +1,33 @@
 from django.db import models
 
 
+class Actor(models.Model):
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
+
+    def __str__(self):
+        return f"{self.first_name} {self.last_name}"
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+
+    def __str__(self):
+        return self.name
+
+
+class CinemaHall(models.Model):
+    name = models.CharField(max_length=255)
+    rows = models.IntegerField()
+    seats_in_row = models.IntegerField()
+
+    def __str__(self):
+        return f"{self.name} (rows: {self.rows}, seats: {self.seats_in_row})"
+
+
 class Movie(models.Model):
     title = models.CharField(max_length=255)
-    description = models.TextField()
+    description = models.CharField(max_length=255)
+    genres = models.ManyToManyField(Genre)
```

Comment:
The `ManyToManyField` for `genres` should have a `related_name` defined. According to the checklist, 'Don't forget to define the `related_name` for `ManyToManyField`.'